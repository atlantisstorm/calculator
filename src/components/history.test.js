import React from 'react';
import { render } from "@testing-library/react";
import History from './history';

describe('History component', () => {
  it('should match snapshot', () => {
    const history = [
      {
        calculation: '5+5',
        result: "10"
      },
      {
        calculation: '5*5',
        result: "25"
      }
    ];
    const { asFragment } = render(
      <History history={ history } />
    );

    expect(asFragment()).toMatchSnapshot();
  });

  it.skip('more test here!', () => {
    // Plus loads of other tests...
  });
});