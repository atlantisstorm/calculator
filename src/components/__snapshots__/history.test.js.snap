// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`History component should match snapshot 1`] = `
<DocumentFragment>
  <div
    class="history"
  >
    <span>
      History
    </span>
    <div>
      <div
        class="item"
      >
        <span>
          5+5
        </span>
        <br />
        <span
          class="result"
        >
          10
        </span>
      </div>
      <div
        class="item"
      >
        <span>
          5*5
        </span>
        <br />
        <span
          class="result"
        >
          25
        </span>
      </div>
    </div>
  </div>
</DocumentFragment>
`;
